### ISSUE TITLE
Pre-defined annotation

### ISSUE THREAD

**Issue by codegenerator01** [lit26/streamlit-image-label#4]
Not really an issue more like a question, is it possible to load in a pre-defined annotation box?

**Comment by lit26** [comment]
Yup, if you have xml file. It will draw the box. Example will be in `img_dir` folder.
